src/main/java/com/study/gamereviewrest/controller/GameController.java

```diff
@@ -0,0 +1,50 @@
+package com.study.gamereviewrest.controller;
+
+import com.study.gamereviewrest.exceptionHangler.GameNotFoundException;
```

Comment:
why is there a package called controller?
did you not read http://www.javapractices.com/topic/TopicAction.do?Id=205